dev_server.py

```diff
@@ -0,0 +1,31 @@
+import argparse
```

Comment:
this is a python version of dev_server.sh

Why? Because if you start a process with a python function in PyCharm all subprocesses are debuggable.